target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -94,31 +88,23 @@ ath79_setup_interfaces()
 		;;
 	"tplink,tl-wr1043nd-v2"|\
 	"tplink,tl-wr1043nd-v3")
-		ucidef_set_interfaces_lan_wan "eth1.1" "eth0.2"
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5:wan" "6@eth0"
 		;;
 	"tplink,tl-wr2543-v1")
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "9@eth0"
 		;;
-	"netgear,wndr3800")
-		ucidef_set_interfaces_lan_wan "eth0" "eth1"
-		ucidef_add_switch "switch0" \
-			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "5u@eth0"
-
-		ucidef_add_switch_attr "switch0" "blinkrate" 2
-		ucidef_add_switch_port_attr "switch0" 1 led 6
-		ucidef_add_switch_port_attr "switch0" 2 led 9
-		ucidef_add_switch_port_attr "switch0" 5 led 2
-		;;
-	"buffalo,wzr-hp-g450h")
+	"tplink,tl-wr740n-v2")
+		ucidef_set_interfaces_wan "eth0"
 		ucidef_add_switch "switch0" \
-			"0@eth0" "2:lan" "3:lan" "4:lan" "5:lan" "1:wan"
+			"0@eth1" "1:lan" "2:lan" "3:lan" "4:lan"
 		;;
-	"phicomm,k2t")
+	"tplink,tl-wr740nd-v4"|\
+	"tplink,tl-wr741nd-v4")
+		ucidef_set_interfaces_wan "eth0"
 		ucidef_add_switch "switch0" \
-			"0@eth0" "3:lan:1" "5:lan:2" "4:wan"
+			"0@eth1" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1"
```

Comment:
You are switching wan and lan eth here, is this by intention?
Would be good to mention this in the commit message. Also would be good to mention other removed ucidif_set_interfaces_*.
If this is already merged, the first question would still be relevant!

Reply:
Sorry I forgot to mention those after changing other things.
WR740N v4/WR741N v4 are using AR9331 and the builtin switch is connected to eth1. So this should be a mistake.

Reply:
So you are saying that you changed it by intention and the new version is correct?
In this case, you should also change the target/linux/ath79/base-files/etc/board.d/01_leds, where the wan led is assigned to eth1 ...

Reply:
Yes.
I'll use a separated commit for this device.